background/background.js

```diff
@@ -335,11 +346,14 @@ const ContextMenu = {
           await createItems();
         }
 
-        // Disable "Open all unread pages in tabs" if all feeds are already
-        // read/visited.
+        // Disable "Open all unread pages in tabs" and "Mark all as read" 
+        // if all feeds are already read/visited.
         browser.menus.update(this.openUnreadItemId, {
           enabled: (await getUnreadChildren(bookmarkId)).length >= 1
         });
+        browser.menus.update(this.markAllReadItemId, {
+          enabled: (await getUnreadChildren(bookmarkId)).length >= 1
+        });
```

Comment:
Can you make this change to avoid fetching the children twice?
```suggestion
        const hasUnreadChildren = (await getUnreadChildren(bookmarkId)).length >= 1;
        // Disable "Open all unread pages in tabs" and "Mark all as read" 
        // if all feeds are already read/visited.
        browser.menus.update(this.openUnreadItemId, {
          enabled: hasUnreadChildren
        });
        browser.menus.update(this.markAllReadItemId, {
          enabled: hasUnreadChildren
        });
```